CMakeLists.txt

```diff
@@ -110,6 +110,7 @@ add_subdirectory(DDCond)
 add_subdirectory(DDAlign)
 add_subdirectory(DDEve)
 add_subdirectory(DDDB)
+add_subdirectory(DDTest)
 
 dd4hep_enable_tests( DDTest )
```

Comment:
Actually build_testing=off fails too. See this issue
https://github.com/AIDASoft/DD4hep/issues/80